CryptoTechReminderSystem/DomainObject/SlackDeveloper.cs

```diff
@@ -0,0 +1,10 @@
+namespace CryptoTechReminderSystem.DomainObject
+{
+    public class SlackDeveloper : Developer
+    {
+        public string Id { get; set; }
+        public string Name { get; set; }
+        public string DisplayName { get; set; }
+        public bool IsBot { get; set; }
```

Comment:
Some properties can be removed(Name, DisplayName, IsBot)

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/9/commits/ecc86dff1db89c95beae2c0287c83e2d39118f78